apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -275,6 +302,7 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
               toggleSwitchAtTheEnd={true}
               labelClassName="text-sm"
               title={t("limit_booking_frequency")}
+              {...bookingLimitsLocked}
```

Comment:
This was removed recently to allow these fields to be unlocked for v1, now with v2 we need them as the team admin can discretely select their state